spots-api/api/get-spots.js

```diff
@@ -1,14 +1,20 @@
 //get-spots.js
 
-module.exports = function() {
+module.exports = function(db) {
 
   // Returns an express req/res callback function
   return function(req, res) {
-    res.json([{
-      id: '12345',
-      location: [[12.34,34.56]],
-      name: 'Starbucks',
-      checkIns: ['user1', 'user2', 'user3']
-    }]);
+    db.listCollections().toArray((err, collections) => {
```

Comment:
This will list collections in the db (like listing tables,  not the actual rows)

Try the below:
```js
db.collection('spots').find({}, (err, spots) => {
  if(err) return res.status(500).json({code: 'DbError', err: err});
  else return res.json(spots);
})
```